website/docs/components/form/checkbox/partials/guidelines/guidelines.md

```diff
@@ -10,6 +10,14 @@
 - When only one choice can be selected, consider [Radio](/components/form/radio/) buttons.
 - When checking or unchecking results in an immediate change, consider [Toggle](/components/form/toggle).
 
+### Indeterminate State
+
+Checkboxes, primarily toggling between checked and unchecked states, offer a third, indeterminate state for indicating a "partially checked" condition in nested structures, manageable not through HTML, but JavaScript.
```

Comment:
This feels a little verbose and technical for the Design Guidelines. Instead, we should lean into providing proper usage guidance for the specific topic. I also suggest adding a few visual examples of proper and improper usage. 

```suggestion
Indeterminate states indicate a "partially checked" condition in nested structures or tables. 
```

Reply:
'_manageable not through HTML, but JavaScript._' unsure of the value of this statement in the guidelines section – I'm tempted to leave it out.

Reply:
@alex-ju also, should we mention the fact that the `indeterminate={{true}}` is an Ember thing (and potentially link to some documentation for it) and not a native HTML feature?

Reply:
If we feel it's important to highlight that this is managed by the component via Ember I can add it [to the code section](https://helios.hashicorp.design/components/form/checkbox?tab=code#indeterminate).

Reply:
Probably yes: as a developer (expecially coming from other frameworks) I would struggle to understand how this indeterminate works (also because is missing the `@` typical of Ember arguments.

Reply:
cool, I'll add a commit to this branch (if that's ok with you, Andrew)

Reply:
Works for me 🙂

Reply:
@didoo added an extra paragraph in 2325b2e; we also touched on this in the API section